sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -520,7 +522,7 @@ case class DescribeTableCommand(table: TableIdentifier, isExtended: Boolean, isF
 
   private def describeSchema(schema: Seq[CatalogColumn], buffer: ArrayBuffer[Row]): Unit = {
     schema.foreach { column =>
-      append(buffer, column.name, column.dataType.toLowerCase, column.comment.orNull)
+      append(buffer, column.name, column.dataType.toLowerCase, column.comment.getOrElse(""))
```

Comment:
I'm not sure whether no comment or empty comment matters here though. Maybe making them consistent makes more sense here.